pkg/collector/pipeline.go

```diff
@@ -49,17 +49,41 @@ func (c nowClock) Now() time.Time {
 var defaultClock nowClock
 
 // Pipeline is a series of processors that should be applied to each report that
-// the collector receives.
+// the collector receives. It uses a fixed number of workers to process the reports
+// and a fixed sized queue that the workers read from. If the queue fills, reports
+// are dropped.
 type Pipeline struct {
```

Comment:
My Go might be a weak, but if you want to prevent someone from instantiating one of these directly (without calling NewPipeline, for example), could you call this "pipeline" instead of "Pipeline"?

Reply:
Maybe, but there's no need to do that. I added to the comment stating that the zero initialized version is not useful.